### DISCUSSION TITLE
アイデア提案

### DISCUSSION THREAD

**Discussion by kenton116** [kenton116/n-development-team#8] [Ideas]
アイデアはこちらでどうぞ

**Comment by fuuchin** [comment]
おはようございます。
野田ゲーみたいなゲームを作れたらなぁと思います。
野田ゲーみたいだと思われる条件

1. **絶対に笑う**
2. 簡単操作
3. 対戦できるようにする
4. 課金できる（戻るボタンはなし）

3はJavaScriptで実装できるのかなーー？
そもそもブラウザで動かすのはできるのかなーー？
